src/PaymentBundle/DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('selector')->defaultValue('sonata.payment.selector.simple')->cannotBeEmpty()->end()
                 ->scalarNode('generator')->defaultValue('sonata.payment.generator.mysql')->cannotBeEmpty()->end()
                 ->arrayNode('transformers')
+                    ->addDefaultsIfNotSet()
```

Comment:
Sorry for the missunderstanding, since travis is green and a second contributor approved this PR, we gonna merge it!

Thanks for your effort 👍